awx/main/middleware.py

```diff
@@ -25,6 +26,14 @@
 perf_logger = logging.getLogger('awx.analytics.performance')
 
 
+class AWXGuidMiddleware(GuidMiddleware):
+
+    def _get_id_from_header(self, request):
+        # never accept the correlation ID from the HTTP response header
+        request.correlation_id = self._generate_guid()
```

Comment:
`django-guid` lets you pass in a specific value in the HTTP request; we don't want this behavior

Reply:
Just wondering why you don't want this behaviour?  
If triggering a job from another API or service that uses a similar correlation ID system, this is a great way to track a request/job/call across multiple systems.